### PR TITLE
Add adcellerant case study

### PR DESCRIPTION
**Pull request by mooreds**
This is a blog post about the adcellerant case study.

### CODE REVIEW

File: site/_posts/2021-04-08-adcellerant-migrated-from-mongodb.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+> "We didn't want to build our own identity service, but if we had, it would look like FusionAuth..." - Jeff
+
+In particular, the application had scaling issues as they added more customers. There were also multiple user data models. These were all stored in a MongoDB database. The lack of a standard user data object made application development more difficult. 
+
+The growing pains of the monolithic application impacted the engineering team timelines, unfortunately. They also made it difficult to deliver on promises to clients.
+
+AdCellerant seamlessly migrated their data from MongoDb to FusionAuth with no impact on users. Thousands of accounts were transferred. The AdCellerant team continues to migrate applications from the homegrown auth system to FusionAuth. 
+
+They've also been able to use FusionAuth for new projects, including building an API to allow external developers to integrate with their flagship product. We'll let James have the final word:
```
Comment (jobannon) [inline site/_posts/2021-04-08-adcellerant-migrated-from-mongodb.md:28]: They've also been able to use FusionAuth for new projects to drive innovation and growth.  One such project involves building an API to allow external developers to integrate with their flagship product.  Lest we sound too boastful, we'll let James have the final word:

**Review by jobannon — COMMENTED** [review]
As usual, just preference items suggested.

Not sure why my local has dealcloser listed?  Maybe it is b/c I need to clean my cache or are we grabbing the wrong files?
image

File: site/_posts/2021-04-08-adcellerant-migrated-from-mongodb.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+James Humphrey, Senior Director of Technology at AdCellerant, and Jeff Fairley, Senior Director of Engineering at AdCellerant, sat down and talked to us about how this growing adtech company is using FusionAuth. 
+
+<!--more-->
+
+We covered the gamut, from the details of the migration of over 100,000 user attributes from a MongoDB database with multiple user data models to the FusionAuth support provided during and after the migration.
+
+Before migrating to FusionAuth, [AdCellerant](https://www.adcellerant.com/) used a homegrown authentication solution. It was an application where authentication logic was intermingled with business logic. This application worked fine initially, but as the company grew, issues arose. 
+
+> "We didn't want to build our own identity service, but if we had, it would look like FusionAuth..." - Jeff
```
[the remark this answers is not in the document]
Comment (mooreds) [reply site/_posts/2021-04-08-adcellerant-migrated-from-mongodb.md:20]: Great call.

File: site/_posts/2021-04-08-adcellerant-migrated-from-mongodb.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+In particular, the application had scaling issues as they added more customers. There were also multiple user data models. These were all stored in a MongoDB database. The lack of a standard user data object made application development more difficult. 
+
+The growing pains of the monolithic application impacted the engineering team timelines, unfortunately. They also made it difficult to deliver on promises to clients.
```
[the remark this answers is not in the document]
Comment (mooreds) [reply site/_posts/2021-04-08-adcellerant-migrated-from-mongodb.md:24]: Hmmm. I don't think so.

File: site/_posts/2021-04-08-adcellerant-migrated-from-mongodb.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+AdCellerant seamlessly migrated their data from MongoDb to FusionAuth with no impact on users. Thousands of accounts were transferred. The AdCellerant team continues to migrate applications from the homegrown auth system to FusionAuth.
```
[the remark this answers is not in the document]
Comment (mooreds) [reply site/_posts/2021-04-08-adcellerant-migrated-from-mongodb.md:26]: Love it.

**Comment by mooreds** [comment]
Thanks for the feedback @jobannon . The reason you saw the old image and title is because I hadn't updated them.

**Pull request merged**